packages/suite/src/utils/suite/transactionReview.ts

```diff
@@ -0,0 +1,23 @@
+import { TranslationKey } from '@suite-common/intl-types';
+import { FormState } from '@suite-common/wallet-types';
+
+export const getTitleOrBtnText = (
```

Comment:
```suggestion
export const getReviewModalActionText = (
```
or something similar

Reply:
https://github.com/everstake/trezor-suite/pull/4/commits/f258205008a78970a6f132c5d995383205820251